### PR TITLE
Show security/privacy tips in the loading screen

### LINKED ISSUE #4139: Include security + privacy messages
**Description:**

We do a great job of including our approach to security and privacy on replay.io and replay.io/security-privacy. We should bring this into the product in a couple of places to help build trust with our users.

I think it can go in a couple of places, but three places where it would feel natural to me would be:

1. https://app.replay.io/browser/new-tab
2. upload screen
3. loading ui 

[figma](https://www.figma.com/file/SSr1ljyzF0lCXTL7BX1Whh/Replay-Design-Doc?node-id=7760%3A48981)

### Here are some examples messages
Frame 952 (1)

### Here's an example of what it could look like in the loading ui. 

I imagine that the cards could be clickable and link to the relevant section of our /security-privacy page.

Uploading Flow (1)

### PR DESCRIPTION
**Pull request by jaril**
Fix #4139.

This adds a slider UI to show security/privacy messaging from Replay while the user is in the loading screen.

This only displays the tips once we have a loading progress bar, for continuity reasons. Extra marks for making it work even before that, so will keep this as a draft til I get that in!
- Added a breakpoint at width=640px so that it only appears when it's bigger than that.
- Current interval is set at ~5s.
- We display the tips in all loading screen as soon as the store is ready.

https://user-images.githubusercontent.com/15959269/141351576-3a22bb1c-5874-4077-83f2-a3686fdc4321.mov

### CODE REVIEW

**Comment by jasonLaster** [comment]
Totally fine to land this and follow up. Would also love to see Nebulas in here 😉

File: src/ui/components/shared/LoadingTip.tsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  {
+    title: "Private by default",
+    description: "We do not view or analyze your replays without your explicit permission.",
+    icon: "zypsy-security-icon3.svg",
+  },
+  {
+    title: "Values aligned",
+    description:
+      "We respect the privacy of everyone who uses our software and do not sell customer data.",
+    icon: "zypsy-security-icon3.svg",
```
Comment (jasonLaster) [inline src/ui/components/shared/LoadingTip.tsx:36]: lets delete the last one. It's redundant.

File: src/ui/components/shared/LoadingTip.tsx [hunk 2]
```diff
@@ -0,0 +1,82 @@
+import classNames from "classnames";
+import React, { useEffect, useRef, useState } from "react";
+import { connect, ConnectedProps } from "react-redux";
+import { actions } from "ui/actions";
+import { selectors } from "ui/reducers";
+import { UIState } from "ui/state";
+
+const TIPS = [
+  {
+    title: "SAML / SSO Integration",
```
Comment (jasonLaster) [inline src/ui/components/shared/LoadingTip.tsx:10]: > SAML 2.0 / SSO Authentication

File: src/ui/components/shared/LoadingTip.tsx [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+  {
+    title: "Encryption at Rest & In-Transit",
+    description:
+      "Replay uses TLS v1.2 in transit and AES 256 at rest so that your data is secure at all times.",
+    icon: "zypsy-security-icon2.svg",
+  },
+  {
+    title: "Intellectual property",
+    description:
+      "We respect the privacy of everyone who uses our software and do not sell customer data.",
```
Comment (jasonLaster) [inline src/ui/components/shared/LoadingTip.tsx:24]: It'd be nice to be able to demonstrate how folks source code and trade secrets are safe with us

> We architected our backend with source code, trade secrets, and personally identifiable information in mind.

Comment (jasonLaster) [reply src/ui/components/shared/LoadingTip.tsx:24]: @jaril here are some new versions
https://www.notion.so/replayio/Security-tips-a77cc563fb4f4b1ca1cfd9b566b0a9f1

**Comment by jaril** [comment]
@jasonLaster Changes are in!

**Review by jasonLaster — APPROVED** [review]

**Pull request merged**